src/main/java/com/gluonhq/substrate/target/IosTargetConfiguration.java

```diff
@@ -113,11 +113,12 @@ public IosTargetConfiguration(ProcessPaths paths, InternalProjectConfiguration c
         if (!Files.exists(internalLlcPath.resolve("llc"))) {
             graalvm22 = false; // and use customLLC
         }
-        List<String> answer = Arrays.asList("-H:CompilerBackend=" + Constants.BACKEND_LLVM,
+        List<String> answer = new ArrayList<>();
+        answer.addAll(Arrays.asList("-H:CompilerBackend=" + Constants.BACKEND_LLVM,
```

Comment:
`Arrays.asList` shouldn't be required here.

Reply:
it's merged. Can you do a follow-up PR on this?